### ISSUE TITLE
Patches ordered by arrival time, rather than patch number

### ISSUE THREAD

**Issue by kuba-moo** [getpatchwork/patchwork#386]
Both UI and JSON API orders patches by date, rather that ID, so clients can't apply patches one by one.

mbox of the entire series is correct, but for build bots is useful to apply each patch individually and build test to check for patches which may break bisection.

See this series as an example: https://patchwork.kernel.org/api/series/390649/
First patch listed is patch 10, because it arrived first.

**Comment by stephenfin** [comment]
I don't think we can solve this in the web UI until we have a proper series view, however, we can certainly sort the embedded list of patches included in `/api/series/{seriesID}` responses by number.

**Comment by daxtens** [comment]
Isn't there a filter that can do that in the API already? order=-id?

**Comment by stephenfin** [comment]
That will apply to the series themselves - not the embedded patches in the response.
